mlir/include/mlir/Dialect/ArmSME/Transforms/CMakeLists.txt

```diff
@@ -2,6 +2,10 @@ set(LLVM_TARGET_DEFINITIONS Passes.td)
 mlir_tablegen(Passes.h.inc -gen-pass-decls -name ArmSME)
 mlir_tablegen(PassesEnums.h.inc -gen-enum-decls)
 mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs)
+mlir_tablegen(Passes.capi.h.inc -gen-pass-capi-header --prefix ArmSME)
+mlir_tablegen(Passes.capi.cpp.inc -gen-pass-capi-impl --prefix ArmSME)
+mlir_tablegen(PassesEnums.h.inc -gen-enum-decls -gen-pass-capi-header --prefix ArmSME)
+mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs -gen-pass-capi-impl --prefix ArmSME)
```

Comment:
I think the CMake errors are becaus there's both:
```
// Likely added since you rebased your PR:
mlir_tablegen(PassesEnums.h.inc -gen-enum-decls)
mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs)
```
And (added in this PR):
```
mlir_tablegen(PassesEnums.h.inc -gen-enum-decls -gen-pass-capi-header --prefix ArmSME)
mlir_tablegen(PassesEnums.cpp.inc -gen-enum-defs -gen-pass-capi-impl --prefix ArmSME)
```

Reply:
I think these need to be merged.

Reply:
Let's try it